src/main/kotlin/at.phatbl.shellexec/ShellExec.kt

```diff
@@ -24,11 +25,11 @@ open class ShellExec: DefaultTask() {
     @InputDirectory
     var workingDir: File = project.projectDir
 
-    @Internal
-    val standardOutput: OutputStream = GradleLogOutputStream(logger, LogLevel.LIFECYCLE)
+    @Input
+    var standardOutput = GradleLogOutputStream(logger, LogLevel.LIFECYCLE, null)
```

Comment:
The `OutputStream` type is actually more important with your change to make this property writable as the caller shouldn't need to extend `GradleLogOutputStream` in order to provide a custom `OutputStream`. `null` won't be necessary here with the default argument for `logFiles` (below).

```suggestion
    var standardOutput: OutputStream = GradleLogOutputStream(logger, LogLevel.LIFECYCLE)
```